src/styles/components/_about-us-introduction.scss

```
@@ -0,0 +1,95 @@
+.about-us-introduction {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: rem(250px);
+
+  @include respond-above('sm') {
+    height: rem(400px);
+  }
+  @include respond-above('md') {
+    height: rem(500px);
+  }
+
+  @include respond-above('lg') {
+    height: rem(600px);
+  }
+  @include respond-above('xl') {
+    height: rem(700px);
+  }
+
+  &__title {
+    width: 58%;
+    height: 12.5%;
+    margin-block-start: 0%;
+
+    color: $dark;
+    -webkit-text-stroke: rem(1px) $dark;
+
+    @include respond-above('md') {
+      font-size: rem(28px);
+    }
+
+    @include respond-above('lg') {
+      font-size: rem(32px);
+    }
+
+    @include respond-above('xl') {
+      width: 45%;
+
+      font-size: rem(36px);
+    }
+
+    font-size: rem(24px);
+    font-weight: bold;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: 1.22;
+    letter-spacing: normal;
+    text-align: left;
+  }
+
+  &__description {
+    transform: translateY(-10%);
+    align-self: flex-end;
+    box-sizing: border-box;
+    width: 55%;
+    height: 55%;
+    padding: 3% 5% 2% 2.5%;
+
+    color: $light;
+    background-color: black;
+
+    @include respond-above('md') {
+      font-size: rem(16px);
+    }
+
+    @include respond-above('lg') {
+      font-size: rem(20px);
+    }
+
+    @include respond-above('xl') {
+      font-size: rem(24px);
+    }
+
+    font-size: rem(14px);
+    font-weight: 200;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: 1.5;
+    letter-spacing: normal;
+    text-align: left;
+  }
+
+  &--background {
+    background: url(../../../public/images/hand-shake.jpg) left 10% top 42%
+        no-repeat,
+      url(../../../public/images/plain-black-rectangle.png) left bottom 40%
+        no-repeat;
+    background-size: 53% auto, 100% 45%;
+  }
```

Would be nice to separate them for readability. But the fact is when change to my suggestion the second box don't work 🤔 
```suggestion
  &--background {
    background: {
      image: url('/images/hand-shake.jpg'), url('/images/plain-black-rectangle');
      repeat: no-repeat;
      position: left 10% top 42%, left bottom 40%;
      size: 53% auto, auto 45%
    }
  }
```

---

After discussing we found both ways have the same result. Anyway, for readability enhancement, I’ll change it to longhand syntax.